nvdaHelper/vbufBackends/gecko_ia2/gecko_ia2.cpp

```diff
@@ -952,7 +952,27 @@ VBufStorage_fieldNode_t* GeckoVBufBackend_t::fillVBuf(
 			// The object has no text, but we do want to render its children.
 			auto [varChildren, accChildRes] = getAccessibleChildren(pacc, 0, childCount);
 			if (S_OK != accChildRes || varChildren.size() == 0) {
-				LOG_ERROR(L"AccessibleChildren failed (count: " << childCount << L"), res: " << accChildRes);
+				std::wstringstream msg;
+				msg << L"AccessibleChildren failed (count: " << childCount << L"), res: " << accChildRes;
+				switch (accChildRes) {
+				case E_NOINTERFACE:
+					msg << L" (E_NOINTERFACE, No such interface supported)";

```

Comment:
I don't know how useful it is to differentiate E_NOINTERFACE from other errors (as opposed to warnings), since one would have to look up the error message for any other error code anyway. That said, I don't have a specific objection.

Reply:
I only included it because we are encountering it now. Using the specific name and description should help recognizing it as the same/known issue.